tests/scripts/run_profiler_regressions.sh

```diff
@@ -5,10 +5,13 @@ source scripts/tools_setup_common.sh
 set -eo pipefail
 
 run_additional_T3000_test(){
+    if [ "$ARCH_NAME" != "wormhole_b0" ]; then
```

Comment:
I believe this would still create an issue like the first version of code.
If future new ARCH_NAMEs are release which was not compatible with this test, we would still encounter the same scenario where this this test is unintentionally executed and 
Since t3000s are running on wormhole_b0 cards, it would be better to explicitly set it as execute for only wormhole_b0 cards.
```
if [ "$ARCH_NAME" = "wormhole_b0" ]; then
    logic
fi
```